contracts/VaultV3.vy

```diff
@@ -19,26 +19,35 @@ interface IStrategy:
    def vault() -> address: view
    
 # EVENTS #
-event Transfer:
-    sender: indexed(address)
-    receiver: indexed(address)
-    value: uint256
+event StrategyAdded: 
+   strategy: indexed(address)
+
+event StrategyRevoked: 
+   strategy: indexed(address)
 
 event Deposit:
-    recipient: indexed(address)
+    fnCaller: indexed(address)
```

Comment:
Should be named `caller`

Reply:
https://github.com/ethereum/EIPs/pull/5184

Reply:
will use sender as it seems to have been approved